tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc

```diff
@@ -623,73 +356,29 @@ StatusOr<std::unique_ptr<Executable>> NVPTXCompiler::RunBackend(
   }
   VLOG(2) << "Libdevice dir = " << libdevice_dir << "\n";
 
-  int cc_major, cc_minor;
-  if (!stream_exec->GetDeviceDescription().cuda_compute_capability(&cc_major,
-                                                                   &cc_minor)) {
-    LOG(WARNING)
-        << "Couldn't get compute capability for device; assuming sm_20.";
-    cc_major = 2;
-    cc_minor = 0;
-  }
-
   string ptx;
   {
-    XLA_SCOPED_LOGGING_TIMER("NVPTXCompiler::RunBackend - CompileToPtx");
+    XLA_SCOPED_LOGGING_TIMER("NVPTXCompiler::InvokeBackend - CompileToPtx");
```

Comment:
NVPTXCompiler::CompileTargetBinary

Reply:
thanks for spotting this. will address this in the next round.